src/content/guides/code-splitting.md

```diff
@@ -294,6 +302,49 @@ __src/index.js__
 ```
 
 
+## Prefecting/Preloading modules
+
+webpack 4.6.0+ adds support for prefetching and preloading.
+
+Using these inline directives while declaring your imports allows webpack to output “Resource Hint” which tells the browser that this resource is probably (prefetch) or definitely (preload) needed for some navigation in the future.
+
+Simple prefetch example can be having a `HomePage` component, which renders a `LoginButton` component which then on demand loads a `LoginModal` component after being clicked.
+
+__LoginButton.js__
+
+```js
+//...
+import(/* webpackPrefetch: true */ "LoginModal");
+
+```
+
+This will result `<link rel="prefetch" href="login-chunk.js">` appended in the HTML page. Which will instruct webpack to prefetch (in browser idle time) this On-Demand-Loaded chunk when the parent chunk finish loading.
```

Comment:
Some small improvements here:

- "This will result **in** ... **being** appended"
- I would change the href to `login-modal-chunk.js`
- No need for period if sentence is going to start with _which_
- on-demand-loaded shouldn't be capitalized
- also, webpack doesn't do the prefetching, it just appends the link element and the rest depends on the browser

This would be the final form of this paragraph.

```markdown
This will result in `<link rel="prefetch" href="login-modal-chunk.js">` being appended in the head of the page, which will instruct the browser to prefetch in idle time the `login-modal-chunk.js` file.

T> webpack will add the prefetch hint once the parent chunk has been loaded.
```